nevergrad/optimization/optimizerlib.py

```diff
@@ -2159,6 +2160,33 @@ def __init__(
 NaiveIsoEMNA = EMNA().set_name("NaiveIsoEMNA", register=True)
 
 
+@registry.register
+class ModCMA(base.Optimizer):
+    def __init__(
+        self, parametrization: IntOrParameter, budget: tp.Optional[int] = None, num_workers: int = 1
+    ) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        from modcma import AskTellCMAES
+
+        self.ModCMA = AskTellCMAES(
```

Comment:
This is an object, it should not be capitalized

```suggestion
        self._modcma = AskTellCMAES(
```